src/07_slices.py

```diff
@@ -12,26 +12,26 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+print(a[1])
 
 # Output the second-to-last element: 9
-print()
+print(a[-2])
 
 # Output the last three elements in the array: [7, 9, 6]
-print()
+print(a[:-4:-1])
```

Comment:
As written this prints `[1, 7, 9]`; I'm pretty sure you know how to do it, so take another look here

Reply:
Sorry, I missed that. Just fixed.
`print(a[-3:])`